spec/decorators/usps_decorator_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+class DummyController < ApplicationController
+  def initialize
```

Comment:
@zachmargolis this approach works, but it feels really hacky to me. Any better suggestions?